src/commands/variations/update.ts

```diff
@@ -65,11 +66,14 @@ export default class UpdateVariation extends UpdateCommand {
         } else {
             selectedVariation = await fetchVariationByKey(this.authToken, this.projectKey, featureKey, args.key)
         }
-
+        const feature = await fetchFeatureByKey(this.authToken, this.projectKey, featureKey)
```

Comment:
could we just do this if we're not using the feature prompt? otherwise we're fetching the feature from the api twice

Reply:
I still need to fetch even if were not using the feature prompt, since the feature prompt doesnt return the full feature. I made a ticket to refactor that because the prompt is being used in several different places

Reply:
Won't the request to fetch the variables associated with the feature already return an error like this if the key is invalid?

Reply:
yeah I think youre right. Pretty sure I got baited by typescript here